.github/CHANGELOG.md

```diff
@@ -346,13 +379,13 @@
   selected or deselected during local execution of tests.
   [(#1633)](https://github.com/PennyLaneAI/pennylane/pull/1633)
 
-* Hamiltonians are now natively supported on the `default.qubit` device if `shots=None`. 
+* Hamiltonians are now natively supported on the `default.qubit` device if `shots=None`.
   This makes VQE workflows a lot faster in some cases.
   [(#1551)](https://github.com/PennyLaneAI/pennylane/pull/1551)
   [(#1596)](https://github.com/PennyLaneAI/pennylane/pull/1596)
```

Comment:
Hmm, this one I would probably move to the 'Faster Hamiltonian' section of 'New Features'. I wonder if @soranjh has any benchmark plots we could include in order to show-off the speed improvement

Reply:
Yes, I have some data. Please let me prepare a clean figure.

Reply:
Looks awesome @soranjh 💪 

Only suggestion is minor - might be good to modify the title to say

> Time per 1 VQE step for H2 on default.qubit

to avoid readers thinking this speedup occurs for hardware devices/finite-shots

Reply:
![image](https://user-images.githubusercontent.com/40344468/133626185-815c0251-e263-48ec-9b0e-4a22055be96e.png)
Another version

Reply:
![image](https://user-images.githubusercontent.com/40344468/133629346-25e9e422-8612-40f3-8f17-015eae49f6a8.png)

Reply:
😍

Reply:
Added the latest plot in https://github.com/PennyLaneAI/pennylane/pull/1671/commits/fa6df8f21f462544bef7a8cbf12903f93b84e5b3 as part of https://github.com/PennyLaneAI/pennylane/pull/1671.